src/wrapper/objects/global_ref.rs

```diff
@@ -41,8 +41,11 @@ impl<'a> From<&'a GlobalRef> for JObject<'a> {
 }
 
 impl GlobalRef {
-    /// Creates a new global reference. This assumes that `NewGlobalRef`
-    /// has already been called.
+    /// Creates a new global reference.
+    ///
+    /// # Safety
+    ///
+    /// This function assumes that `NewGlobalRef` has already been called.
```

Comment:
Where is `NewGlobalRef` function in Rust API?

Reply:
Specified it's JNI function. The same for other mentions.
P.S. I like the idea it says in comments where to get expected safe value.